src/components/BattleScreen/BattleScreen.js

```diff
@@ -48,11 +48,9 @@ function BattleScreen({player1, player2}){
   useEffect(() => {
       createBattle();
     },[createBattle]);
-    
-    // if (loading) return 'Loading...';
+
```

Comment:
I think we might have to look into how we implemented this call as it is currently double sending the creatBattle mutation query but I'm not sure how to fix it right now without accidentally sending it into a loop